.github/workflows/build-android.yaml

```
@@ -0,0 +1,63 @@
+name: Build Android
+
+on:
+  workflow_dispatch:
+    inputs:
+      ndkVersion:
+        description: Android NDK Version (e.g., r26b)
+        required: true
+        default: r26b
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - .mailmap
+      - '**.md'
+      - AUTHORS
+      - doc/**
+      - .github/**
+      - '!.github/workflows/build-android.yml'
```

```suggestion
      - '!.github/workflows/build-android.yaml'
```
Or better, rename the file to be consistent (`git mv .github/workflows/build-android.yaml .github/workflows/build-android.yml`)